tests/common/common_python_vec.py

```diff
@@ -23,6 +23,7 @@
 from collections import defaultdict
 from string import Template
 from itertools import product
+from math import ceil, floor
```

Comment:
While you are here, you could sort the imports. :-)

Reply:
Sure, I went ahead and ordered them  based on the module name. Hope that's the preferred way of doing it.